src/vg/civcraft/mc/civchat2/listeners/MercuryMessageListener.java

```diff
@@ -0,0 +1,48 @@
+package vg.civcraft.mc.civchat2.listeners;
+
+import java.util.UUID;
+
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.scheduler.BukkitRunnable;
+
+import vg.civcraft.mc.civchat2.CivChat2;
+import vg.civcraft.mc.mercury.events.AsyncPluginBroadcastMessageEvent;
+import vg.civcraft.mc.namelayer.NameAPI;
+import vg.civcraft.mc.namelayer.NameLayerPlugin;
+
+public class MercuryMessageListener extends BukkitRunnable implements Listener {
```

Comment:
Also on a side note.  BukkitRunnable is depricated and they suggest just using a regular Runnable